src/components/Facility/FacilityHome.tsx

```diff
@@ -334,7 +344,7 @@ export const FacilityHome = ({ facilityId }: Props) => {
 
               {facilityData?.description && (
                 <Card>
-                  <CardContent className="mt-4">
+                  <CardContent className="mt-4 text-justify">
```

Comment:
i don't think we should justify markdown content

```suggestion
                  <CardContent className="mt-4">
```

Reply:
![image](https://github.com/user-attachments/assets/7d07173f-1fdc-4b93-891f-eea46c500030)

I felt this content justification looks better

![image](https://github.com/user-attachments/assets/2dd5d84e-2156-4f9d-a8d3-b877d675f5a3)

Reply:
But ya on moblie it looks a bit weird, removing it.

Reply:
okiez